huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_upgrading_minor_version_test.go

```diff
@@ -0,0 +1,47 @@
+package rds
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccRdsUpgradingMinorVersion_basic(t *testing.T) {
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckRdsInstanceId(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      nil,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccRdsUpgradingMinorVersion_basic(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("huaweicloud_rds_upgrading_minor_version.test", "job_id"),
+					resource.TestCheckOutput("is_delayed_is_useful", "true"),
+				),
+			},
+		},
+	})
+}
+
+func testAccRdsUpgradingMinorVersion_basic() string {
+	return fmt.Sprintf(`
+resource "huaweicloud_rds_upgrading_minor_version" "test" {
+  instance_id = "%s"
+}
+
+resource "huaweicloud_rds_upgrading_minor_version" "is_delayed_filter" {
+  instance_id = "%s"
+  is_delayed  = true
+}
+
+output "is_delayed_is_useful" {
+  value = length(data.huaweicloud_rds_upgrading_minor_version.is_delayed_filter.job_id) > 0
+}
```

Comment:
should be deleted, it is only used in data source, it is used to check whether the query param is effective